packages/template-retail-react-app/app/pages/checkout/index.jsx

```diff
@@ -10,13 +10,13 @@ import {Alert, AlertIcon, Box, Button, Container, Grid, GridItem, Stack} from '@
 import useNavigation from '../../hooks/use-navigation'
 import {CheckoutProvider, useCheckout} from './util/checkout-context'
 import ContactInfo from './partials/contact-info'
-import ShippingAddress from './partials/shipping-address'
-import ShippingOptions from './partials/shipping-options'
+// import ShippingAddress from './partials/shipping-address'
+// import ShippingOptions from './partials/shipping-options'
 import useCustomer from '../../commerce-api/hooks/useCustomer'
```

Comment:
Are we still using this? I see that there is a Component named CheckoutContainer uses these two hooks to render skeleton. We should replace them with our hooks

Reply:
good call, replaced.